connector/src/yang/connector/settings.py

```diff
@@ -1,5 +1,9 @@
+try:
+    from pyats.datastructures import AttrDict
+except ImportError:
+    class AttrDict:
+        pass
 
-from pyats.datastructures import AttrDict
 
```

Comment:
Better to define AttrDict like this, no try/except needed and the settings can be accessed as required.
```
class AttrDict(dict):

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.__dict__ = self

    def __repr__(self):
        return '%s(%s)' % (self.__class__.__name__,
                           super(AttrDict, self).__repr__())
```

Reply:
AttrDict seems unnecessary.  Why not define Settings as above?
```
class Settings(dict):

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.__dict__ = self
        # Default number of lines to log to screen
        self.NETCONF_SCREEN_LOGGING_MAX_LINES = 40
        # Enable XML formatting by default
        self.NETCONF_LOGGING_FORMAT_XML = True

    def __repr__(self):
        return '%s(%s)' % (self.__class__.__name__,
                        super(AttrDict, self).__repr__())
```

Reply:
Let's use this, dont need the repr

```python
class Settings(dict):

    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
        self.__dict__ = self
        # Default number of lines to log to screen
        self.NETCONF_SCREEN_LOGGING_MAX_LINES = 40
        # Enable XML formatting by default
        self.NETCONF_LOGGING_FORMAT_XML = True
```

Reply:
Done